### ISSUE TITLE
Train ViT-UNet on synthetic data (cloud GPU)

### ISSUE THREAD

**Issue by ltrgoddard** [data-desk-eco/burnoff#46]
## Context

The ViT-UNet model architecture and training pipeline are implemented on the `methane-vit-unet` branch. Local proof-of-concept training verified the pipeline works (loss decreases, no MPS crashes), but the model needs proper training on a cloud GPU to achieve useful accuracy.

## Task

Train the model on synthetic data as a first milestone before real S2 data.

### Steps

1. Provision a cloud GPU (Lambda Cloud or RunPod, A100-80GB recommended, ~$1-2/hr)
2. Clone the repo, checkout `methane-vit-unet`, `cd methane/train && uv sync`
3. Run training:
   ```bash
   uv run python train.py --synthetic --epochs 10 --train-len 100000 --batch-size 64
   ```
4. Monitor metrics — target: **F1 > 0.5** on synthetic validation set after 10 epochs
5. If F1 plateaus, try:
   - Increasing `--train-len` to 500K or 1M
   - Adjusting learning rate (`--lr 5e-4`)
   - The scheduler auto-reduces LR after 10 stale batches

### Success criteria

- val F1 > 0.5 on synthetic data
- val FPR < 0.1% pixel-wise
- Checkpoint saved at `checkpoints/best.pt`

### Notes

- Model is 96M params, 366MB fp32 — needs ~8GB VRAM at batch size 64
- Training uses BCEDiceLoss to handle class imbalance
- Paper used 1.235M samples, 10 epochs, Adam lr=1e-3

## Branch

`methane-vit-unet`
